<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI Tạo Câu Hỏi Trắc Nghiệm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* Màu nền chính */
    QMainWindow {
        background-color: #f0f0f0;
    }

    /* Text Edit */
    QTextEdit {
        background-color: #ffffff;
        border: 2px solid #cccccc;
        border-radius: 10px;
        padding: 10px;
        font-size: 14px;
        color: #333333;
    }

    /* Nút bấm */
    QPushButton {
        background-color: #4CAF50;
        border: none;
        border-radius: 10px;
        color: white;
        padding: 10px 20px;
        font-size: 16px;
        font-weight: bold;
    }

    QPushButton:hover {
        background-color: #45a049;
    }

    QPushButton:pressed {
        background-color: #3d8b40;
    }

    /* Group Box */
    QGroupBox {
        background-color: #ffffff;
        border: 2px solid #cccccc;
        border-radius: 10px;
        padding: 15px;
        font-size: 14px;
        color: #333333;
        margin-top: 10px;
    }

    QGroupBox::title {
        subcontrol-origin: margin;
        subcontrol-position: top left;
        padding: 0 5px;
        font-size: 16px;
        font-weight: bold;
        color: #4CAF50;
    }

    /* Label */
    QLabel {
        font-size: 16px;
        color: #333333;
    }

    QLabel#titleLabel {
        font-size: 18px;
        font-weight: bold;
        color: #4CAF50;
    }

    /* Line Edit */
    QLineEdit {
        background-color: #ffffff;
        border: 2px solid #cccccc;
        border-radius: 10px;
        padding: 8px;
        font-size: 14px;
        color: #333333;
    }

    /* Spin Box */
    QSpinBox {
        background-color: #ffffff;
        border: 2px solid #cccccc;
        border-radius: 10px;
        padding: 8px;
        font-size: 14px;
        color: #333333;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Nhập nội dung để tạo câu hỏi trắc nghiệm</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="inputTextEdit">
      <property name="maximumHeight">
       <number>150</number>
      </property>
      <property name="placeholderText">
       <string>Nhập nội dung tại đây...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="numQuestionsLabel">
        <property name="text">
         <string>Số câu hỏi:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numQuestionsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Tạo Câu Hỏi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="questionsGroupBox">
      <property name="title">
       <string>Câu Hỏi Trắc Nghiệm</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="questionsTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="answersGroupBox">
      <property name="title">
       <string>Nhập đáp án của bạn</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="answersTextEdit">
         <property name="maximumHeight">
          <number>100</number>
         </property>
         <property name="placeholderText">
          <string>Nhập đáp án của bạn (ví dụ: 1-A, 2-B, 3-C, 4-D, 5-A)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Nộp Bài</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="scoreLabel">
      <property name="text">
       <string>Điểm: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>